File: solutions/javascript/raindrops/1/raindrops.js
export const convert = (num) => {
  let result = '';
  const threeCheck = num % 3 === 0;
  const fiveCheck = num % 5 === 0;
  const sevenCheck = num % 7 === 0;
  
  if (threeCheck) {
    result += 'Pling';
    if (fiveCheck) {
      result += 'Plang';
      if (sevenCheck) {
        result += 'Plong';
      }
    } else if (sevenCheck) {
      result += 'Plong';
    }
  } else if (fiveCheck) {
    result += 'Plang';
    if (sevenCheck) {
      result += 'Plong';
    }
  } else if (sevenCheck) {
    result += 'Plong';
  } else {
    result += num;
  }

  return result;
};

File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    result = ""
    if number % 3 == 0:
        result += "Pling"
    if number % 5 == 0:
        result += "Plang"
    if number % 7 == 0:
        result += "Plong"
    if not number % 3 == 0 and not number % 5 == 0 and not number % 7 == 0:
        result += str(number)

    return result
